tests/src/Search/Web/ExampleWebTest.php

```diff
@@ -33,7 +33,7 @@ public function testElasticSearchIndex()
                 ],
```

Comment:
tests/src/Search/Api/Elasticsearch/InterviewElasticsearchTest.php should need an update? It uses the minimum and complete JSONs so there should be one with an actual image being returned and checked?